### PR TITLE
Fix Rubyflake.time method returning very wrong time

### PR DESCRIPTION
**Pull request by joallard**
I also added a few standard things, and set the stage for testing.

### CODE REVIEW

File: lib/rubyflake.rb [hunk 1]
```diff
@@ -27,14 +27,14 @@ def generate
   # Extract the time from an ID.
   #
   def time(id)
-    Time.at(((id >> Rubyflake::FLAKE_TIMESTAMP_SHIFT) + @epoch) / 1000)
+    Time.at(((id >> Rubyflake::FLAKE_TIMESTAMP_SHIFT) / 1000) + @epoch)
```
Comment (joallard) [inline lib/rubyflake.rb]: "Oops!"

**Comment by mindware** [comment]
Any objections to this being merged, @fubelly?

File: lib/rubyflake.rb (hunk 1 above)
Comment (mtthgn) [inline lib/rubyflake.rb]: oops indeed!

**Comment by mtthgn** [comment]
Hey everyone, sorry for _such a slow response_! Merging this now.

Also curious, what's the purpose of extracting the time from the id? Someone else added it to the gem. I never really understood the value since the IDs are sortable by time created anyway...

File: lib/rubyflake.rb (hunk 1 above)
Comment (mindware) [inline lib/rubyflake.rb:30]: Looks like tagging you worked. :-) Consider adding Joallard as a contributor, we could get this show on the road.

**Comment by mindware** [comment]
Thanks for the merge. Do we bump the version number?
(Not really sure about the usage of extracting the time to be honest, we should ask the original contributor or see @joallard input)

**Comment by joallard** [comment]
PR etiquette is not to change the version number and let the maintainer do it, but I think you're right and this qualifies as a bugfix bump. Maybe we could get some other features in before a minor release.

**Pull request merged**